circuit_puzzles/programs/statutes_mutation.clsp

```diff
@@ -51,27 +74,27 @@
   )
 
   (list
-    ; we're updating if indexes match and it's not announcements
-    (if (= mutation_index CUSTOM_ANNOUNCEMENTS_MUTATION_INDEX)
-      ; this is a vote for announcements
+    ; update statutes unless we output custom conditions
+    (if (= mutation_index CUSTOM_CONDITIONS_MUTATION_INDEX)
+      ; custom conditions case
       STATUTES
-      ; update statutes
+      ; update statute corresp to given index
       (mutate-list mutation_index mutation_value STATUTES 0)
     )
-    ; prices staying the same
+    ; price info stays the same
     PRICE_INFO
-    ; don't change the rates
+    ; discount factors stay the same
     PAST_CUMULATIVE_STABILITY_FEE_DF
     PAST_CUMULATIVE_INTEREST_DF
+    ; price update counter stays the same
     PRICE_UPDATE_COUNTER
 
-    ;  updating statutes, assert permission
+    ; updating statutes
     (assert
       (size_b32 crt_inner_puzzle_hash)
-      ; -1 is for announcements, rest are for statutes
-      (> mutation_index -2)
-      (> 37 mutation_index) ; there are 37 statutes, no more
-      (if (= mutation_index CUSTOM_ANNOUNCEMENTS_MUTATION_INDEX)
+      (> mutation_index -2) ; -1 for custom conditions, >= 0 for statutes
+      (> 37 mutation_index)
```

Comment:
what's wrong with "; there are 37 statutes, no more" comment?

Reply:
making this a named constant in statutes_utils.clib